editor/modes/visual-editor/block.js

```diff
@@ -229,7 +241,26 @@ class VisualEditorBlock extends wp.element.Component {
 						</div>
 					</CSSTransitionGroup>
 				}
-				<div onKeyPress={ this.maybeStartTyping }>
+				{ this.props.isFirstSelected && (
```

Comment:
Yeah. I think I forgot to remove this after I saw it clashes with the selector... Any naming conventions here?